lib/cloudformation.py

```diff
@@ -465,14 +465,19 @@ def create(self, session, wait = True):
                 raise Exception("Could not determine argument '{}'".format(argument["ParameterKey"]))
 
         client = session.client('cloudformation')
-        response = client.create_stack(
-            StackName = self.stack_name,
-            TemplateBody = self._create_template(),
-            Parameters = self.arguments,
-            Tags = [
-                {"Key": "Commit", "Value": utils.get_commit()}
-            ]
-        )
+
+        try:
+            response = client.create_stack(
+                StackName = self.stack_name,
+                TemplateBody = self._create_template(),
+                Parameters = self.arguments,
+                Tags = [
+                    {"Key": "Commit", "Value": utils.get_commit()}
+                ]
+            )
+        except client.exceptions.AlreadyExistsException:
+            print('{} already exists, aborting.'.format(self.stack_name))
+            return False
```

Comment:
Maybe return `None` so if needed the caller can tell between an actual failure and no change? A `None` value is still considered `False` in an `if` statement, so I don't think it should change any of the caller logic.

Reply:
Adjusted to return None.